packages/@aws-cdk-testing/framework-integ/test/aws-stepfunctions/test/integ.state-machine-disable-xray.ts

```diff
@@ -0,0 +1,25 @@
+import * as iam from 'aws-cdk-lib/aws-iam';
+import * as cdk from 'aws-cdk-lib';
+import * as sfn from 'aws-cdk-lib/aws-stepfunctions';
+/*
+ * Stack verification steps:
+ *
+ * -- aws stepfunctions describe-state-machine --state-machine-arn <stack-output> has a status of `ACTIVE`
+ * -- aws iam get-role-policy --role-name <role-name> --policy-name <policy-name> has all actions mapped to respective resources.
+ */
+const app = new cdk.App();
+const stack = new cdk.Stack(app, 'aws-stepfunctions-integ');
+
+// const role = new iam.Role(stack, 'Role', {
+//   assumedBy: new iam.ServicePrincipal('lambda.amazonaws.com'),
+// });
+
+const stateMachine = new sfn.StateMachine(stack, 'StateMachine', {
+  comment: 'a super cool state machine',
+  tracingEnabled: false,
+});
+
+// stateMachine.grantRead(role);
+// stateMachine.grant(role, 'states:SendTaskSuccess');
+
+app.synth();
```

Comment:
a snapshot is needed for this integration test file - which is why some checks are failing. this snapshot will confirm that these changes are deployable to an AWS account. is it possible to add that snapshot?